security/spoofing/README.md

```diff
@@ -0,0 +1,110 @@
+# Workshop Spoofing
+
+✔️ Learn how to create an SMTP server
+
+✔️ Discover how to spoof an email
+
+✔️ Create an evil proxy
+
+## Setup
+                        !!!! CHECK YOUR EMAIL !!!
+     The server IP will be given to you by mail in your Epitech account.
+          Attached to this IP you will receive an rsa private key.
+
+Before starting the workshop, you need to create an account on [PoC Security Platform](https://challs.poc-innovation.com/).
+
+Then, download the private key received by mail and save it as `id_rsa` in your home directory.</br>
+Don't forget to change the permissions of the file to 600: `chmod 600 id_rsa`</br>
+Once you have the private key, you can connect to the server using the following command:
+`ssh -i id_rsa root@<IP>`</br>
+
+The next step is to install prerequisites for the workshop:
+```
+sudo apt update && sudo apt upgrade && sudo apt dist-upgrade
+
+sudo apt install mailutils && sudo apt install sendemail
+
+sudo dpkg-reconfigure postfix
+```
+
+
+## Step 1: SMTP server
+
+- Setup relay restrictions
+- Configure hostname and network
+- Specify send-only parameters
+- Set spoofed masquerade
+
+To get the flag, send your postfix configuration in `/etc/postfix/main.cf` to a PoC Security administrator.</br>
+You can send it in `workshop-spoofing@proton.me` with your Epitech email address with the following subject: `SMTP server`.</br>
```

Comment:
Same for Epitech content

Are the credentials of the `workshop-spoofing@proton.me` email kept somewhere for future usage ?